packages/utils/build-tools/src/configureMetro.ts

```diff
@@ -66,29 +67,30 @@ export function addPlatformMetroConfig(platform: PlatformValue, base: any = {}):
  *
  * @param options - metro configuration options
  */
-export function configureMetro(optionsToMerge?: object) {
+export async function configureMetro(optionsToMerge?: object) {
+  const {
+    resolver: { sourceExts, assetExts },
+  } = await getDefaultConfig();
+
   const options = {
     // WatchFolders is only needed due to the yarn workspace layout of node_modules, we need to watch the symlinked locations separately
-    watchFolders: [
-      path.resolve(findGitRoot(), 'node_modules'),
-      ...getPackageInfo()
-        .dependencies()
-        .paths()
-    ],
+    watchFolders: [path.resolve(findGitRoot(), 'node_modules'), ...getPackageInfo().dependencies().paths()],
     resolver: {
-      resolverMainFields: ['react-native', 'browser', 'main']
+      resolverMainFields: ['react-native', 'browser', 'main'],
+      assetExts: assetExts.filter((ext) => ext !== 'svg'),
+      sourceExts: [...sourceExts, 'svg'],
```

Comment:
remove trailing ,.  Do you have prettier settings turned on for vs-code?

Reply:
Yes I do have it on. Are we opposed to its opinions? 😆

I couldn't find a way to turn it off, clicking on formatting in the status bar doesn't do anything and an extension to disable it that I downloaded doesn't work very well. How is everyone else doing it?

Reply:
I see prettier is just an extension that is installed by default. I just uninstalled it.

Reply:
Ok something else is formatting it upon save. Prettier is uninstalled, formatOnSave is disabled but it still formats whenever I ctrl+s.

Reply:
Oh I found it. The checked in .vscode/settings.json has this line set:

  "editor.formatOnSave": true,

It's overriding vscode global settings.